### PR TITLE
docs(maintainers): initial maintainers playbook

### PR DESCRIPTION
**Pull request by heitorlessa**
## Summary

Issue number: #1076

### Changes

> Please provide a summary of what's being changed

Document describing who are the maintainers, processes to maintain this project, etc.

### User experience

> Please share what the user experience looks like before and after this change

## Checklist

If your change doesn't seem to apply, please leave them unchecked.

* [x] [Meet tenets criteria](https://awslabs.github.io/aws-lambda-powertools-python/#tenets)
* [x] I have performed a self-review of my this change
* [ ] Changes are tested
* [x] Changes are documented
* [x] PR title follows [conventional commit semantics](https://github.com/awslabs/aws-lambda-powertools-python/blob/376ec0a2ac0d2a40e0af5717bef42ff84ca0d1b9/.github/semantic.yml#L2)

<details>
<summary>Is this a breaking change?</summary>

**RFC issue number**:

Checklist:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

</details>

## Acknowledgment

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Disclaimer**: We value your time and bandwidth. As such, any pull requests created on non-triaged issues might not be successful.

-----
[View rendered MAINTAINERS.md](https://github.com/heitorlessa/aws-lambda-powertools-python/blob/docs/new-maintainers/MAINTAINERS.md)

### CODE REVIEW

File: MAINTAINERS.md
Comment (mploski) [inline MAINTAINERS.md:147]: Should we add one additional question about usage pattern? Something like:
`What would be preferred usage if solution can be used in different ways?` Think about our discussion about e2e testing - We want to put suggestion into RFC when to use this feature,so maybe we can make this question explicit in this section?

File: MAINTAINERS.md
Comment (mploski) [inline MAINTAINERS.md:240]: Maybe we could add additional paragraph  to mantainers responsibility to constantly evaluate repeating cases and update common scenarios if they feel all group would benefit from it. I think it make sense since we want this section to evolve over time as we grow and we experience more :-)

File: MAINTAINERS.md
Comment (heitorlessa) [reply MAINTAINERS.md:147]: great idea!

Comment (heitorlessa) [reply MAINTAINERS.md:147]: Added: `Does the proposal address the use case? If so, is the recommended usage explicit?`

Let me know if that suffices.

File: MAINTAINERS.md
Comment (heitorlessa) [reply MAINTAINERS.md:243]: Great idea too. Added them at the top:

> Be aware of recurring ambiguous situations and [document them](#common-scenarios) to help your fellow maintainers.

Added this paragraph within Common scenarios section to reemphasize their purpose and defer to maintainer's judgement.

> These are recurring ambiguous situations that new and existing maintainers may encounter. They serve as guidance. It is up to each maintainer to follow, adjust, or handle in a different manner as long as [our conduct is consistent](#uphold-code-of-conduct)

Let me know if it suffices.

**Comment by heitorlessa** [comment]
@mploski addressed your comments. Let me know if they suffice, and we can merge it.

**Review by mploski — APPROVED** [review]

**Pull request merged**
